server/controllers/browser.js

```diff
@@ -0,0 +1,15 @@
+const { Visitor } = require('../models');
+
+module.exports = {
+  all: function(req, res) {
+    Visitor
+      .findAll()
```

Comment:
This is a potential efficiency pitfall. There would definitely be users/visitors with the same browser on record, just fetching all of them and then looping through to get the browser data is inefficient, we need to find a way we can make a query to the `visitors collection` such that it returns all visitors but with the `unique browserType`.